src/screens/TopicMemberScreen/index.js

```diff
@@ -88,7 +88,7 @@ const TopicMemberScreen = () => {
     return () => {
       scrollY.removeListener(listener);
     };
-  }, [scrollY]);
+  }, []);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The removal of `scrollY` from the dependency array in the `useEffect` hook is a good change. This prevents unnecessary re-registrations of the listener, as `scrollY` is a ref and its identity doesn't change during the component's lifecycle.
```diff
-  }, [scrollY]);
+  }, []);
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->